src/Surfnet/ServiceProviderDashboard/Domain/Entity/ManageEntity.php

```diff
@@ -18,6 +18,8 @@
 
 namespace Surfnet\ServiceProviderDashboard\Domain\Entity;
 
+use SebastianBergmann\Type\UnknownType;
```

Comment:
This dependency should not be used here in the domain! It seems to be a dev-leftover. So no man overboard